packages/pure/src/device/device.test.ts

```diff
@@ -4,20 +4,22 @@
  */
 
 import mockSerialPort from "../mock-serial-port"
-import PureNode, { MuditaDevice, RequestConfig } from "../index"
-import {
+import PureNode, {
+  MuditaDevice,
   DeviceEventName,
   Endpoint,
   Method,
+  RequestConfig,
   ResponseStatus,
-} from "./device.types"
+} from "../index"
 
 let device: MuditaDevice
 
 beforeEach(async (done) => {
   mockSerialPort()
 
   const devices = await PureNode.getDevices()
+  console.log("devices: ", devices)
```

Comment:
Let's remove it

Reply:
https://github.com/mudita/mudita-center/pull/697/commits/32a9172dd4cf1699b3d0d7babaa727fdce4e8e10